bot/utils/time.py

```diff
@@ -111,3 +111,33 @@ def parse_rfc1123(stamp: str) -> datetime.datetime:
 def format_infraction(timestamp: str) -> str:
     """Format an infraction timestamp to a more readable ISO 8601 format."""
     return dateutil.parser.isoparse(timestamp).strftime(INFRACTION_FORMAT)
+
+
+def get_duration_from_expiry(
+    expiry: str = None,
+    date_from: datetime.datetime = None,
+    max_units: int = 2
+) -> Optional[str]:
+    """
+    Returns a human-readable version of the  the duration between datetime.utcnow() and an expiry.
+
+    Unlike the original function, this function will force the precision to be 'seconds' by not passing it.
+    max_units specifies the maximum number of units of time to include (e.g. 1 may include days but not hours).
+    By default, max_units is 2
+
+    :param expiry: A string. If not passed in, will early return a None ( Permanent infraction ).
+    :param date_from: A datetime.datetime object. If not passed in, will use datetime.utcnow().
+    :param parts: An int, to show how many parts will be returned ( year - month or year - month - week - day ...).
```

Comment:
This style for documenting parameters is not used in our code base. Instead, they are either part of the paragraphs or, in cases with many arguments, as a bullet point list. I don't remember exactly but surely you'll find some examples if you look around elsewhere in the code base.

Reply:
Hmm, I was actually looking at the function right above, `wait_until` and I saw this style and thought that it will automatically build the doc from this. I will simply remove this part, it should be clear enough from the type hint and the docstring.